<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModeManagerDlg</class>
 <widget class="QDialog" name="ModeManagerDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mode Management</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comboBox_mode"/>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_add">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="ipponboard.qrc">
         <normaloff>:/res/icons/add.png</normaloff>:/res/icons/add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_remove">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="ipponboard.qrc">
         <normaloff>:/res/icons/delete.png</normaloff>:/res/icons/delete.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_title"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Subtitle</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_subtitle"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_8">
       <property name="toolTip">
        <string>The fight duration in seconds</string>
       </property>
       <property name="text">
        <string>Template</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBox_template"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Weights</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="lineEdit_weights">
         <property name="frame">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_doubleWeights">
         <property name="toolTip">
          <string>Indicates that all weights will be used two times</string>
         </property>
         <property name="text">
          <string>Weights count twice</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string># rounds</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QSpinBox" name="spinBox_rounds">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_fightsPerRound">
         <property name="text">
          <string>FIGHTS_PER_ROUND</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="toolTip">
        <string>The fight duration</string>
       </property>
       <property name="text">
        <string>Fight Time</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QSpinBox" name="spinBox_fightTimeMinutes">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>720</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>min</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_fightTimeSeconds">
         <property name="suffix">
          <string/>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>59</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>sec</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_timeOverrides">
         <property name="toolTip">
          <string>Enables to specify overrides for the fight time for certain categories. E.g. &quot;U12:120;U15:180;U18:240&quot;</string>
         </property>
         <property name="text">
          <string>Use overrides</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="lineEdit_timeOverrides">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>The overrides should be something like this:
&quot;U12:120;U15:180;U18:240&quot;
(&lt;identifier&gt;:&lt;time in seconds&gt;;&lt;identifier&gt;...)
Note: the time must be specified in seconds alone</string>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_9">
       <property name="toolTip">
        <string>The fight duration in seconds</string>
       </property>
       <property name="text">
        <string>Options</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QCheckBox" name="checkBox_2013Rules">
       <property name="toolTip">
        <string>Use the offical IJF rules of 2013</string>
       </property>
       <property name="text">
        <string>Use rules of 2013</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QCheckBox" name="checkBox_autoAdjustPoints">
       <property name="toolTip">
        <string>Automatically add points during Osae-komi</string>
       </property>
       <property name="text">
        <string>Auto adjust points</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QCheckBox" name="checkBox_allSubscoresCount">
       <property name="toolTip">
        <string>Sub-scores are valued even if the fight is lost</string>
       </property>
       <property name="text">
        <string>All sub-scores count</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBox_mode</tabstop>
  <tabstop>toolButton_add</tabstop>
  <tabstop>toolButton_remove</tabstop>
  <tabstop>lineEdit_title</tabstop>
  <tabstop>lineEdit_subtitle</tabstop>
  <tabstop>comboBox_template</tabstop>
  <tabstop>lineEdit_weights</tabstop>
  <tabstop>checkBox_doubleWeights</tabstop>
  <tabstop>spinBox_rounds</tabstop>
  <tabstop>spinBox_fightTimeSeconds</tabstop>
  <tabstop>lineEdit_timeOverrides</tabstop>
  <tabstop>checkBox_2013Rules</tabstop>
  <tabstop>checkBox_autoAdjustPoints</tabstop>
  <tabstop>checkBox_allSubscoresCount</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="ipponboard.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ModeManagerDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ModeManagerDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
